styles/index.css

```diff
@@ -93,6 +88,19 @@ ul {
   height: 100%;
 }
 
+.navbar__icon {
+  fill: #e85a4f;
+}
+
+.navbar__icon--active {
```

Comment:
The term is very speaking, but according to BEM, it should be "navbar__icon-active" (as you have already done below).